### DISCUSSION TITLE
Cant Activate Plugin Preview Link Auto Redirect

### DISCUSSION THREAD

**Discussion by LuckyLiGat** [YOURLS/YOURLS#3456] [Q&A]
Cant Activate Plugin Preview Link Auto Redirect

hurmm error 500

only preview link manual..
how to make it auto..

all qr preview cant activate

**Comment by dgw** [comment]
Error 500 only tells us "something went wrong". Consult the [troubleshooting guide](https://docs.yourls.org/guide/troubleshooting/first-steps.html).
